frontend/ui/data/keyboardlayouts/de_keyboard.lua

```diff
@@ -0,0 +1,9 @@
+-- Start with the english keyboard layout (deep copy, to not alter it)
+local de_keyboard = require("util").tableDeepCopy(require("ui/data/keyboardlayouts/en_keyboard"))
+
+local keys = de_keyboard.keys
+
+keys[1][6][1], keys[3][2][1] = keys[3][2][1], keys[1][6][1] -- Z <-> Y
+keys[1][6][2], keys[3][2][2] = keys[3][2][2], keys[1][6][2] -- z <-> y
+
+return de_keyboard
```

Comment:
`\n` at end of file please :)
(not super important, but github complains, and I sometimes had to fight with git when updating such files).